data/datastore/src/main/proto/com/eblan/launcher/data/datastore/proto/home/text_color.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Buf (1.66.1)

[error] Files must have a package defined.

(PACKAGE_DEFINED)